.scalafmt.conf

```diff
@@ -0,0 +1,26 @@
+version = 3.8.3
+runner.dialect = scala212
+maxColumn = 120
+project.git = true
+
+# http://docs.scala-lang.org/style/scaladoc.html recommends the JavaDoc style.
+# scala/scala is written that way too https://github.com/scala/scala/blob/v2.12.2/src/library/scala/Predef.scala
+docstrings.style = Asterisk
+
+# This also seems more idiomatic to include whitespace in import x.{ yyy }
+spaces.inImportCurlyBraces = true
+
+align.tokens."+" = [
+  {
+    code   = "%"
+    owners = [
+      { regex = "Term.ApplyInfix" }
+    ]
+  },
+  {
+    code   = "%%"
+    owners = [
+      { regex = "Term.ApplyInfix" }
+    ]
+  }
+]
```

Comment:
do we really want this? It's nonstandard, IMHO it doesn't help readability that much, and makes diffs bigger than necessary...

Reply:
I mean not neccessarily, its just that as I said most sbt plugins already use this config (I happen to maintain quite a few).

Do you feel strongly about this? I personally find it easier to read but in cases like this I defer to consistency

Reply:
I'd say do whatever makes you happy to contribute to open source :)

Reply:
@raboof I will merge as is so we can progress forward, if you feel strongly about this change we can always update it later

Reply:
> Do you feel strongly about this?

No, otherwise I wouldn't have approved the PR :)